modules/home-manager/eza.nix

```
@@ -0,0 +1,22 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  inherit (config.catppuccin) sources;
+
+  cfg = config.catppuccin.eza;
+  theme = "${sources.eza}/themes/${cfg.flavor}/catppuccin-${cfg.flavor}-${cfg.accent}.yml";
```

by default we copy the contents of the `themes` dir.
```suggestion
  theme = "${sources.eza}/${cfg.flavor}/catppuccin-${cfg.flavor}-${cfg.accent}.yml";
```